web/src/main/java/com/google/accompanist/web/WebView.kt

```diff
@@ -180,9 +189,10 @@ class WebViewState(webContent: WebContent) {
     var content by mutableStateOf<WebContent>(webContent)
 
     /**
-     * Whether the WebView is currently loading data in its main frame
+     * Whether the WebView is currently [Started] loading data in its main frame (along with
+     * progress) or the data loading has [Finished]. See [LoadingState]
      */
-    var isLoading: Boolean by mutableStateOf(false)
```

Comment:
Let's keep this just as a computed property to make it easier to upgrade for people already using it. Probably something like

```
val isLoading: Boolean
   get() = loadingState !is Finished

Reply:
Yep, I had considered this, but wanted to put it up for discussion first